rskj-core/src/integrationTest/java/pte/PteIntegrationTest.java

```diff
@@ -0,0 +1,270 @@
+
+                results.put(i, response);
+                Thread.sleep(500);
+            }
+
+            completableFuture.complete(results);
+            return null;
+        });
+
+        return completableFuture;
+    }
+
+    private Response sendBulkTransactions(
+            String addressFrom1, String addressTo1,
+            String addressFrom2, String addressTo2,
+            String addressFrom3, String addressTo3,
+            String addressFrom4, String addressTo4) throws IOException {
+
+        String content = "[\n" +
+                "{\n" +
+                "    \"jsonrpc\": \"2.0\",\n" +
+                "    \"method\": \"eth_sendTransaction\",\n" +
+                "    \"id\": 1,\n" +
+                "    \"params\": [{\n" +
+                "        \"from\": \"" + addressFrom1 + "\",\n" +
+                "        \"to\": \"" + addressTo1 + "\",\n" +
+                "        \"gas\": \"0x9C40\",\n" +
+                "        \"gasPrice\": \"0x10\",\n" +
+                "        \"value\": \"0x500\"\n" +
+                "    }]\n" +
+                "},\n" +
+                "{\n" +
+                "    \"jsonrpc\": \"2.0\",\n" +
+                "    \"method\": \"eth_sendTransaction\",\n" +
+                "    \"id\": 1,\n" +
+                "    \"params\": [{\n" +
+                "        \"from\": \"" + addressFrom2 + "\",\n" +
+                "        \"to\": \"" + addressTo2 + "\",\n" +
+                "        \"gas\": \"0x9C40\",\n" +
+                "        \"gasPrice\": \"0x10\",\n" +
+                "        \"value\": \"0x500\"\n" +
+                "    }]\n" +
+                "},\n" +
+                "{\n" +
+                "    \"jsonrpc\": \"2.0\",\n" +
+                "    \"method\": \"eth_sendTransaction\",\n" +
+                "    \"id\": 1,\n" +
+                "    \"params\": [{\n" +
+                "        \"from\": \"" + addressFrom3 + "\",\n" +
+                "        \"to\": \"" + addressTo3 + "\",\n" +
+                "        \"gas\": \"0x9C40\",\n" +
+                "        \"gasPrice\": \"0x10\",\n" +
+                "        \"value\": \"0x500\"\n" +
+                "    }]\n" +
+                "},\n" +
+                "{\n" +
+                "    \"jsonrpc\": \"2.0\",\n" +
+                "    \"method\": \"eth_sendTransaction\",\n" +
+                "    \"id\": 1,\n" +
+                "    \"params\": [{\n" +
+                "        \"from\": \"" + addressFrom4 + "\",\n" +
+                "        \"to\": \"" + addressTo4 + "\",\n" +
+                "        \"gas\": \"0x9C40\",\n" +
+                "        \"gasPrice\": \"0x10\",\n" +
+                "        \"value\": \"0x500\"\n" +
+                "    }]\n" +
+                "}\n" +
+                "]";
+
+        return OkHttpClientTestFixture.sendJsonRpcMessage(content, RPC_PORT);
+    }
```

Comment:
suggestion:

Same from above, migrate this method to the class `OkHttpClientTestFixture` so we can start to have more and more rpc methods there to be re-used.

Reply:
Will do!